docs/course_materials/exercises/07_Exceptions/homework.md

```diff
@@ -0,0 +1,24 @@
+# Exceptions - HomeWork 
+
+### Exc1.py
```

Comment:
the convention of the file is lower Can you choose a different name to the file? Can you choose a different name to the file and change it to lower case?